src/test/java/com/marizueva/laboratory/hw4/testdata/UsedInTestTerms.java

```diff
@@ -0,0 +1,20 @@
+package com.marizueva.laboratory.hw4.testdata;
+
+public class UsedInTestTerms {
```

Comment:
Class should be `final` and has `private` constructor

Reply:
fixed